templates/includes/sidebar.html

```diff
@@ -0,0 +1,9 @@
+<!-- templates/includes/sidebar.html -->
+<div class="sidebar">
+  <ul>
+      <li><a href="#">Home page</a></li>
+      <li><a href="#">Manufacturers</a></li>
+      <li><a href="#">Cars</a></li>
+      <li><a href="#">Drivers</a></li>
```

Comment:
The href attributes of the anchor tags are currently set to '#', which means they don't lead anywhere. You should replace '#' with the actual URLs of the pages.